.github/scripts/end2end/configs/keycloak_config.json

```diff
@@ -19,7 +19,7 @@
       "subGroups": []
     }
   ],
-  "defaultRoles" : [ "uma_authorization", "StorageManager", "offline_access" ],
+  "defaultRoles" : [ "uma_authorization", "offline_access" ],
```

Comment:
leave a comment for reviewers
`StorageManager` should not be set as defaultRole for keycloak users, but it didn't affect existing keycloak users, because existing keycloak users overwrite their roles. but it does affect new added keycloak users, so need to fix here